client/src/components/JurisdictionAdmin/BatchRoundDataEntry.tsx

```diff
@@ -93,17 +120,22 @@ const BatchResultsForm = ({
     jurisdictionId,
     roundId
   )
-  const { register, handleSubmit, watch, formState, errors } = useForm<{
-    [choiceId: string]: number
-  }>({
+  const { register, handleSubmit, watch, formState, errors } = useForm<
+    IBatchResults
+  >({
     defaultValues:
-      batch.results ||
-      Object.fromEntries(contest.choices.map(c => [c.id, undefined])),
+      batch.resultTallySheets.length === 1
+        ? batch.resultTallySheets[0].results
+        : undefined,
```

Comment:
I realized the form doesn't actually need an empty `IBatchResults` object - `undefined` works as well and seems a bit simpler